### DISCUSSION TITLE
curve problem for corsair h150i elite capellix

### DISCUSSION THREAD

**Discussion by malibu303** [liquidctl/liquidctl#612] [Q&A]
As you can see in the screenshot, the first speed set command worked successfully.

![Curve screenshot](https://github.com/liquidctl/liquidctl/assets/18628062/72a3fabe-8666-4ce3-9bce-469bca6ad40b)

All other (curve) commands do not work as described here.
[https://github.com/liquidctl/liquidctl/blob/main/docs/corsair-platinum-pro-xt-guide.md#programming-the-fan-speeds](url)

What am I doing wrong? Please help.

**Comment by dragonshardz** [comment]
It's odd that this doesn't work, since the docs and PR which say the functionality is on GitHub is 3 months old and the latest release - which should have included the fan curve function - is from this month.

**Reply by Bob25zi** [reply]
I have unluckily the same issue with my Corsair icue h150i capellix xt- AIO cooler and Linux Mint system.

**Reply by skip-pacheco** [reply]
I am having the same problem with Corsair Commander ST.  Debian 12 came with version 1.12.1.  I also got the same error after manually installing 1.13.0.

I tried with cloning the latest snapshot from github (v1.14.0dev89+g4f62259) and get a different error.
ERROR: Corsair Commander ST (broken): unexpected error: IndexError('index out of range')

I would be happy to help test any fixes for this.

**Comment by malibu303** [comment]
I bought an air cooler
